src/content/projects/Templeapp.yaml

```diff
@@ -0,0 +1,25 @@
+title: "Temple App"
+members:
+  - user: abhijit
+    role: Maintainer
+  - user: rajendra
+    role: Maintainer
+  - user: yash
+    role: Maintainer
+  - user: ajay
+    role: Developer
+  - user: gopika
+    role: Developer
+  - user: shivangi
+    role: Developer
+
+slug: "Temple App"
+tagline: "Temple App is an Android app aimed for managing the finances of temples"
+description:
+  Temple App is an Android app aimed for managing the finances of temples
+cover: ./covers/Templeapp.png
+gallery:
+  - ./gallery/Templeapp-ss-1.png
+links:
+  github: https://github.com/amfoss/TempleApp
+  irc: https://gitter.im/amfoss/TempleApp?utm_source=share-link&utm_medium=link&utm_campaign=share-link
```

Comment:
```suggestion
  Glitter: https://gitter.im/amfoss/TempleApp?utm_source=share-link&utm_medium=link&utm_campaign=share-link
```

Reply:
Either Change `irc` to Glitter 
Or 
Add Correct IRC Channel URL `#amfoss-android`
https://webchat.freenode.net/?channels=%23amfoss-android

Reply:
Glitter won't show the icon, nor does Gitter, The name irc would show a common message icon so I guess that would work.

Reply:
Then don't point glitter in IRC